src/main/java/ch/njol/skript/lang/Variable.java

```diff
@@ -55,21 +70,12 @@
 import org.skriptlang.skript.lang.script.Script;
 import org.skriptlang.skript.lang.script.ScriptWarning;
 
-import java.lang.reflect.Array;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Map.Entry;
-import java.util.NoSuchElementException;
-import java.util.TreeMap;
-
 public class Variable<T> implements Expression<T> {
 
 	private final static String SINGLE_SEPARATOR_CHAR = ":";
 	public final static String SEPARATOR = SINGLE_SEPARATOR_CHAR + SINGLE_SEPARATOR_CHAR;
 	public final static String LOCAL_VARIABLE_TOKEN = "_";
+	private static final char[] reservedTokens = {'~', '.', '+', '$', '!', '&', '^', '*', '-'};
```

Comment:
Is there any concern about these contents being modified (even via reflection potentially)

Reply:
I don't think this is a particularly high-security risk 😔